dbos/context.py

```diff
@@ -39,6 +40,9 @@ class TracedAttributes(TypedDict, total=False):
     applicationID: Optional[str]
     applicationVersion: Optional[str]
     executorID: Optional[str]
+    authenticatedUser: Optional[str]
+    authenticatedUserRoles: Optional[str]
+    authenticatedUserAssumedRole: Optional[str]
```

Comment:
This casing is a standard for tracing?  (vs. python, where it sticks out like a sore thumb?)

Reply:
It needs to be the same as in TS